zebra/zebra_vty.c

```diff
@@ -61,12 +61,19 @@
 
 extern int allow_delete;
 
+/* context to manage dumps in multiple tables or vrfs */
+struct dump_ctx {
```

Comment:
I'm not just sure if this is the right naming. What others think?

Reply:
display_context ?

Reply:
something like `route_show_ctx` or is this struct designed for kinda _generic_ usage in the future (not only for show ip route)?

Reply:
The struct is indeed for the `show ip route` family of commands (all commands that will eventually invoke `do_show_route_helper()`). `route_show_ctx` sound good to me.

Reply:
I think no more.

Reply:
great, thanks!